packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -298,7 +337,11 @@ const SettingsSidebarContainer = ({
                               textClassNames="px-3 text-emphasis font-medium text-sm"
                               disableChevron
                             />
-                            {(team.role === MembershipRole.OWNER || team.role === MembershipRole.ADMIN) && (
+                            {(team.role === MembershipRole.OWNER ||
+                              team.role === MembershipRole.ADMIN ||
+                              // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                              // @ts-ignore this exists wtf?
+                              (team.isOrgAdmin && team.isOrgAdmin)) && (
```

Comment:
I think it's time to abstract this.

```suggestion
                            {checkIfShouldDisplay(team) && (
```